### DISCUSSION TITLE
im using amd  RX 580 2048sp gpu, what should i do?

### DISCUSSION THREAD

**Discussion by Lewis1210-sudo** [lllyasviel/Fooocus#3545] [General]
![Screenshot 2024-08-20 020250](https://github.com/user-attachments/assets/b8939b5a-220c-4095-baed-da508878823d)

**Comment by Rogala** [comment]
Buy an Nvidia graphics card

**Comment by lgwjames** [comment]
get a egpu like me

**Comment by lgwjames** [comment]
![WhatsApp Image 2024-08-25 at 11 25 41_2383bf38](https://github.com/user-attachments/assets/6b2ae301-30ee-43f9-9646-afd4b694ee54)
